src/Core/AuthenticationHelpers/ClientRoleHeaderAuthenticationMiddleware.cs

```diff
@@ -193,7 +198,8 @@ private static string ResolveConfiguredAuthNScheme(string? configuredProviderNam
             return SimulatorAuthenticationDefaults.AUTHENTICATIONSCHEME;
         }
         else if (string.Equals(configuredProviderName, SupportedAuthNProviders.AZURE_AD, StringComparison.OrdinalIgnoreCase) ||
-            string.Equals(configuredProviderName, SupportedAuthNProviders.ENTRA_ID, StringComparison.OrdinalIgnoreCase))
+            string.Equals(configuredProviderName, SupportedAuthNProviders.ENTRA_ID, StringComparison.OrdinalIgnoreCase) ||
+            string.Equals(configuredProviderName, SupportedAuthNProviders.OAUTH, StringComparison.OrdinalIgnoreCase))
```

Comment:
Lets remove OAUTH as the synonym for AzureAD too. Only EntraID should be the synonym for AzureAD.